ferrocene/doc/user-manual/src/rustfmt/usage.rst

```diff
@@ -0,0 +1,30 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Using ``rustfmt``
+=================
+
+To format a single file, invoke ``rustfmt`` like this:
```

Comment:
We should mention that this also formats all the child modules, so doing `rustfmt src/main.rs` / `rustfmt src/lib.rs` formats the whole project.

Reply:
Didn't know that either. I thought it really just formats the files you state explicitly and that the "formatting the whole project" was a feature of `cargo fmt`.

Reply:
same